src/Microsoft.Azure.WebJobs.Extensions.Storage/Blobs/Listeners/BlobListener.cs

```diff
@@ -6,31 +6,52 @@
 using System.Threading.Tasks;
 using Microsoft.Azure.WebJobs.Host.Listeners;
 using Microsoft.Azure.WebJobs.Host.Scale;
+using Microsoft.Azure.Storage.Blob;
+using Microsoft.Extensions.Logging;
 
 namespace Microsoft.Azure.WebJobs.Host.Blobs.Listeners
 {
     internal sealed class BlobListener : IListener, IScaleMonitorProvider
     {
         private readonly ISharedListener _sharedListener;
+        private readonly ILogger<BlobListener> _logger;
 
         private bool _started;
         private bool _disposed;
+        private Lazy<string> _details;
 
-        public BlobListener(ISharedListener sharedListener)
+        // for mock test purposes only
+        internal BlobListener(ISharedListener sharedListener)
         {
             _sharedListener = sharedListener;
         }
 
+        public BlobListener(ISharedListener sharedListener, CloudBlobContainer container, ILoggerFactory loggerFactory)
+        {
+            _sharedListener = sharedListener;
+            _details = new Lazy<string>(() => $"blob container={container.Name}, storage account name={container.ServiceClient.GetAccountName()}");
```

Comment:
why do we use `Lazy` here?

Reply:
also would be useful to log full[ blob name pattern](https://docs.microsoft.com/en-us/azure/azure-functions/functions-bindings-storage-blob-trigger?tabs=in-process%2Cextensionv5&pivots=programming-language-csharp#blob-name-patterns) from `IBlobPathSource`

Reply:
I saw the `Lazy` pattern in other logging PRs (for example: https://github.com/Azure/azure-sdk-for-net/pull/25566/files). I'm not sure what it accomplished there either, but I just copied that example

Reply:
Removed `Lazy` and added logging the full blob name pattern